aws-apigateway/scripts/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
This is copied from `awsx` right? If it works well for both we can take a ticket to lift this up to ci-mgmt and share.

Reply:
Yes, it is indeed from `awsx`. Let me file this ticket.